Makefile

```diff
@@ -54,6 +54,8 @@ IMG ?= $(IMAGE_TAG_BASE):$(VERSION)
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
 ENVTEST_K8S_VERSION = 1.23
 
+OPERATOR_SDK_VERSION ?= 1.24.0
```

Comment:
We are already o 1.25.0 https://github.com/kiegroup/kogito-serverless-operator/blob/main/CONTRIBUTING.md#prerequisites

Reply:
This is super wrong then, 'cause the files are mentioning 1.24.0. I'll update, thanks!

Reply:
isn't better go with 1.24.1?

Reply:
seems it needs to be 1.25.0
I don't remember if the required versions are in the README, if so it needs to be updated as well.